### DISCUSSION TITLE
Sentiment Analysis in Cryptocurrency Trading (Project 239)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#75] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 239](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Sentiment%20Analysis%20in%20Cryptocurrency%20Trading/README.md)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by billbot-web** [comment]
Hi! My name is Billy Hawkins and I'm a student at Cal Poly, San Luis Obispo. I'll be working on this project and am interested in collaborating with others.

**Reply by AR0OK1ePro** [reply]
Hi, I am a student at NTU Singapore and interested in working and collaborating on this project, would you consider collaborating with me?

**Reply by billbot-web** [reply]
Hey, thanks for reaching out. I am definitely interested in collaborating. Whats the best way to reach you?

**Reply by AR0OK1ePro** [reply]
How about using WhatsApp?

**Reply by billbot-web** [reply]
Yes that will work. Send me an email to billyhawkins45@gmail.com and I'll give you my number.

**Comment by leo10manyam** [comment]
Hey everyone!

Thanks for checking out the MATLAB and Simulink Challenge projects! I’m K Subramanyam, a student from Bapuji Institute of Engineering and Technology, Bangalore, and I’m excited to start working on Fluid Flow Simulation Using Physics-Informed Neural Networks.

I’d love to connect and collaborate with anyone who’s interested in the same topic or has cool ideas to share. Let’s learn from each other and build something awesome!

**Comment by mayank609** [comment]
Hi, I’m a student at IIT Jodhpur and excited to work on the cryptocurrency price prediction using tweet sentiment project. Looking forward to collaborating with others!

**Reply by lkjmo123** [reply]
Hey, I am an Engineering student at the Hong Kong University of Science and Technology and I am interested in working on this project, would you be down to collaborate?

**Comment by omugaowino** [comment]
Hello,am working on this project.
Thanks,
Best Regards: 
Owino Omuga
